### ISSUE TITLE
ParseFilter visibility

### ISSUE THREAD

**Issue by vip32** [pdevito3/QueryKit#4]
Please make ParseFilter<T> public instead of internal, so it can he used more broadly. PR?

**Comment by pdevito3** [comment]
How else would you use it if it was exposed? Hesitant to expose it and confuse people, but if there's a good use case I'll consider it.

**Comment by vip32** [comment]
To get to the expression outside of a dbcontext/dbset construct.

It lets the users of your lib use the functionality in a broader way by
providing the expressions itself. That way they can also be used outside of
EF.

I know you focus on EF, however this query > expression translation is very
interesting in a broader
context.

**Comment by pdevito3** [comment]
Fair enough. Feel free to submit a PR or I'll try and get it in next release.

**Comment by pdevito3** [comment]
FYI, i just published v0.4.0 which includes this
